### ISSUE TITLE
Build works on OSX but dnu restore fails (System.Resources.MissingManifestResourceException)

### ISSUE THREAD

**Issue by calwi** [aspnet/dnx#1983]
I'm trying to test a nightly build of Mono 4.3 to build DNX, ultimately to see if they have fixed an issue with SqlClient/EF7, but am getting the error below when I run `dnu restore`. Essentially it relates to this issue https://github.com/aspnet/Identity/issues/403, linked here https://github.com/aspnet/Identity/issues/423 and raised on mono as https://github.com/mono/mono/pull/1773

If I try to switch back to Windows and VS2015, *.resx file properties doesn't show any options to set as embedded resource or anything anymore, and I've tried to put it in the project.json file as a resource. I see there is another version of this file in Microsoft.Framework.Runtime, along with a generated Designer.cs file. PackageManager doesn't have this, but it doesn't work either way.

Looking for someone to help shed some light on this? Thanks.

```
$dnu restore
Microsoft .NET Development Utility Mono-x86-1.0.0-t150529025402

----------
System.Resources.MissingManifestResourceException: Could not find any resources appropriate for the specified culture or the neutral culture.  Make sure "Microsoft.Framework.PackageManager.NuGet.NuGetResources.resources" was correctly embedded or linked into assembly "Microsoft.Framework.PackageManager" at compile time, or that all the satellite assemblies required are loadable and fully signed.
  at System.Resources.ManifestBasedResourceGroveler.HandleResourceStreamMissing (System.String fileName) <0x1f38e98 + 0x0024f> in <filename unknown>:0 
  at System.Resources.ManifestBasedResourceGroveler.GrovelForResourceSet (System.Globalization.CultureInfo culture, System.Collections.Generic.Dictionary`2 localResourceSets, Boolean tryParents, Boolean createIfNotExists, System.Threading.StackCrawlMark& stackMark) <0x1f37a18 + 0x00247> in <filename unknown>:0 
  at System.Resources.ResourceManager.InternalGetResourceSet (System.Globalization.CultureInfo requestedCulture, Boolean createIfNotExists, Boolean tryParents, System.Threading.StackCrawlMark& stackMark) <0x1f376a0 + 0x0021d> in <filename unknown>:0 
  at System.Resources.ResourceManager.InternalGetResourceSet (System.Globalization.CultureInfo culture, Boolean createIfNotExists, Boolean tryParents) <0x1f37650 + 0x0003f> in <filename unknown>:0 
  at System.Resources.ResourceManager.GetString (System.String name, System.Globalization.CultureInfo culture) <0x1f36e00 + 0x00141> in <filename unknown>:0 
  at NuGet.LocalizedResourceManager.GetString (System.String resourceName) <0x1f36c38 + 0x00046> in <filename unknown>:0 
  at NuGet.PackageSourceBuilder.CreateSourceProvider (ISettings settings) <0x1f35938 + 0x00053> in <filename unknown>:0 
  at Microsoft.Framework.PackageManager.RestoreCommand.ReadSettings (System.String solutionDirectory) <0x1f07ff8 + 0x00077> in <filename unknown>:0 
  at Microsoft.Framework.PackageManager.RestoreCommand+<Execute>d__76.MoveNext () <0x1f06738 + 0x0055f> in <filename unknown>:0 
----------
Restore failed
Could not find any resources appropriate for the specified culture or the neutral culture.  Make sure "Microsoft.Framework.PackageManager.NuGet.NuGetResources.resources" was correctly embedded or linked into assembly "Microsoft.Framework.PackageManager" at compile time, or that all the satellite assemblies required are loadable and fully signed.
```

**Comment by davidfowl** [comment]
Seems like you have custom build of the dnx? Do you see the problem for builds coming from myget?

**Comment by calwi** [comment]
Hi David, originally I was trying to build everything from MSFT, coreclr, corefx, etc but ran into too many problems, because I just wanted to try and get VSCode working. In the end, I just went with all the defaults, and so ran into this problem with EF7/SqlClient.

So I figured, maybe as Mono was a little more mature, especially on OS X, I'd stick with it, so I was talking to one of them on IRC about this problem, and he suggested trying to recompile Mono nightly build, and then I thought well I would also need to recompile DNX to run on that, rather than the using brew , which always wants to do it's own thing. (I was trying avoid using brew entirely at the beginning, but was forced to succumb)

I had seen and grabbed a nupkg for coreclr-darwin from myget dev branch, but nothing worked with that, and I've not seen anything else on there? Not quite sure what you mean?

So, I suppose, I'm only just getting started with this and it's early days. My main aim at the moment is trying to get a working EF7 to SQL 2014 going on OSX, rather than anything else.
